utils/api_auth.py

```diff
@@ -4,7 +4,7 @@
 from pathlib import Path
 import dotenv
 
-env_file = next((Path(__file__).parent.parent / "env").glob("*.env"))
+env_file = next((Path(__file__).parent.parent / "env").glob(".env*"))
```

Comment:
doesn't this cause issues if there are multiple env files present in the folder? It seems like it will randomly choose 1.